dev/Repeater/SelectionModel.cpp

```diff
@@ -647,7 +647,11 @@ void SelectionModel::SelectWithPathImpl(const winrt::IndexPath& index, bool sele
         // If we unselect something, raise event any way, otherwise changedSelection is false
         bool changedSelection = false;
 
-        if (m_singleSelect)
+        // We only need to clear selection by walking the data structure from the beginning when we are in single selection mode
+        // and want to select something.
```

Comment:
Nit: layout. Something you could do:

```suggestion
        // We only need to clear selection by walking the data structure from the beginning when:
        // - we are in single selection mode and 
        // - want to select something.
```